src/Types/CurrencyTypes.php

```diff
@@ -25,7 +25,7 @@
 
 class CurrencyTypes
 {
-    /** @var array<string, string> */
+    /** @var array<string, int> */
     protected static $currencies = [
             'AED' => '784',
```

Comment:
Just need to change all the values now
```php
            'AED' => 784,
            ...
```

Reply:
This will show PHP error "Invalid octal literals for some values like 008"

Reply:
fixed